data/text.js

```diff
@@ -5,7 +5,7 @@ exports.BattleText = {
 		tieBattle: "Tie between [TRAINER] and [TRAINER]!",
 
 		pokemon: "[NICKNAME]",
-		opposingPokemon: "the opposing [NICKNAME]",
+		opposingPokemon: "The opposing [NICKNAME]",
```

Comment:
This part is definitely wrong, yes. It should be lowercase in the middle of a sentence; there's other code which capitalizes this when it's at the beginning of a sentence.